embed.py

```diff
@@ -31,4 +31,35 @@ def captainlist():
   embed.add_field(name= 'Dollars:', value = captain_dollars_string)
   return embed
 
-  
\ No newline at end of file
+def player_info(message):
+  players = db['players']
+  message_parts = message.content.split()
+  preferences = ['Pos 1: ', 'Pos 2: ', 'Pos 3: ', 'Pos 4: ', 'Pos 5: ', 'Hero Drafter: ']
+  pref_string = '\n'.join(preferences)
+  name = message_parts[1]
+  for player in players:
+    if player['name'] == name:
+      ratings = [
+        player['pos1'],
+        player['pos2'],
+        player['pos3'],
+        player['pos4'],
+        player['pos5'],
+        player['hero_drafter'], 
+        ]
```

Comment:
unindent this so it has same indent level as `ratings`

Reply:
Unintended bracket